app/views/notification_mailer/announcement_email.html.erb

```diff
@@ -1,7 +1,13 @@
-<%= "#{t('mailer.new_announcement')} #{@announcement_details}:" %>
-<p>
-  <%= "#{t('mailer.author')}: #{@announcement.announcer.name}" %>
-  <br>
-  <%= "#{t('mailer.text')}: " %>
-  <%= @announcement.details.html_safe %>
-</p>
\ No newline at end of file
+<p class="h1">
+  Mitteilungstitel
```

Comment:
@fosterfarrell9 Imo this would be a nice change, but its too much for me at the moment since the model / db / form etc. has to be changed